srcpkgs/uriparser/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'uriparser'
+pkgname=uriparser
+version=0.8.6
+revision=1
+build_style=gnu-configure
+hostmakedepends="pkg-config doxygen graphviz qt5-tools"
+makedepends="cpptest-devel"
+short_desc="RFC 3986 compliant URI parsing and handling library"
+maintainer="mobinmob <mobinmob@disroot.org>"
+license="BSD-3-Clause"
+homepage="https://uriparser.github.io/"
+distfiles="https://github.com/${pkgname}/${pkgname}/releases/download/${pkgname}-${version}/${pkgname}-${version}.tar.bz2"
+checksum=0709a7e572417db763f0356250d91686c19a64ab48e9da9f5a1e8055dc2a4a54
+
+post_install() {
+	vlicense COPYING
+}
+
+uriparser-devel_package() {
+	depends="$makedepends"
```

Comment:
should also depend on the main package with version and revision restrictions

${soourcepkg}-${version}_${revision}

Reply:
Fixed.